crates/hyperqueue/src/server/event/log/stream.rs

```diff
@@ -54,17 +54,17 @@ async fn streaming_process(
     loop {
         tokio::select! {
             _ = flush_fut.tick() => {
-                writer.flush().await?;
+                writer.flush()?;
```

Comment:
Why is this no longer async? That could block the main HQ thread. Especially if we're writing to LUSTRE...

Reply:
No, all operations over the file that runs in its own thread. And it actually do only one thing at the time. It is async only because we need to communicate via async queues form main thread.